types/ws/index.d.ts

```diff
@@ -143,12 +144,6 @@ declare namespace WebSocket {
         family?: number;
         checkServerIdentity?(servername: string, cert: CertMeta): boolean;
         rejectUnauthorized?: boolean;
-        passphrase?: string;
```

Comment:
These keys I removed are all defined in `SecureContextOptions`. The difference here is that they are defined correctly in `SecureContextOptions`